tests/test_pagination.py

```diff
@@ -331,6 +348,62 @@ def test_relay_cursor_sorted(client, data):
     }
 
 
+def test_relay_cursor_sorted_none_desc(client, data_with_nulls):
+    response = client.get("/relay_cursor_widgets?sort=-name")
+    assert_response(
+        response, 200, [{"id": "8", "name": None}, {"id": "7", "name": None}]
```

Comment:
i know this the default behavior with psql, but i find this sort behavior so odd, and am i little concerned our users won't understand it. e.g will they expect that "no exam type" is sorted last when sorting alphabetically. @sloria do you have any opinions on whether our users will expect this?

Reply:
we should probably not ask and act surprised later i think